src/services/notificationHandler.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+const NotificationHandler = require('../lib/notificationHandlerWebsocket')
```

Comment:
I was using the more general English meaning. What new name do you suggest? `notificationSender` ?